src/classes/PermanentFileSystem.ts

```diff
@@ -161,6 +192,12 @@ export class PermanentFileSystem {
       return cachedFolder;
     }
 
+    if (isRootPath(requestedPath)
```

Comment:
It's probably better to just check `isItemPath` than check all three of these explicitly.